src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig

```diff
@@ -3,6 +3,13 @@
 {% block root %}
     {% set menu_items %}
         {% for item in item.children %}
+            {% if loop.first %}
+                {%- do item.setExtras(item.extras|merge({
```

Comment:
Not sure whether we really want to do it always and do not provide an option to opt-out.

Reply:
Turns out we want this behaviour. Personally, I'm not convinced, but... 🙂